components/textfield/stories/textfield.stories.js

```diff
@@ -80,16 +76,36 @@ export default {
 		isRequired,
 		isReadOnly,
 		isLoading,
-		pattern: {
-			name: "Pattern",
+		pattern: { table: { disable: true } },
+		value: { table: { disable: true } },
+		helpText: {
+			name: "Help text (description)",
 			type: { name: "string" },
+			control: { type: "text" },
 			table: {
 				type: { summary: "string" },
-				category: "Component",
+				category: "Content",
 			},
-			control: "text",
 		},
-		value: { table: { disable: true } },
+		displayCounter: {
+			name: "Show counter",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Advanced",
+			},
+			control: "boolean",
+		},
+		characterCount: {
+			name: "Character counter",
+			type: { name: "number" },
+			table: {
+				type: { summary: "boolean" },
```

Comment:
Is this supposed to be `number` instead of `boolean`?

Reply:
`number`